core/trino-server/src/main/provisio/trino.xml

```diff
@@ -170,6 +170,12 @@
         </artifact>
     </artifactSet>
 
+    <artifactSet to="plugin/http-event-listener">
```

Comment:
Should this be `plugin/mysql-event-listener` or are they combined on purpose?

Reply:
Good catch :-)